application-xpoll-ui/src/main/resources/XPollCode/XPollTemplateProvider.xml

```diff
@@ -19,8 +19,8 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.3" reference="XPollCode.XPollTemplateProvider" locale="">
-  <web>XPollCode</web>
+<xwikidoc version="1.3" reference="XPoll.XPollCode.XPollTemplateProvider" locale="">
+  <web>XPoll.XPollCode</web>
```

Comment:
Same as above. The page reference doesn't match the page location on the file system.